bokeh/server/blaze/views.py

```diff
@@ -35,8 +34,8 @@ def _make_range(r):
 @mbsbp.route("/render/<docid>/<datasourceid>/<glyphid>", methods=['GET', 'POST'])
 def render(docid, datasourceid, glyphid):
     #load bokeh document
-    bokehuser = bokeh_app.authentication.current_user()
-    request_username = bokehuser.username
+    #bokehuser = bokeh_app.authentication.current_user()
+    #request_username = bokehuser.username
     clientdoc = bokeh_app.backbone_storage.get_document(docid)
```

Comment:
I came across quite a few of these. I commented them, rather then removing, because I thought we might want to use it at some point. Or you can let me know which ones we can just get rid of.